examples/python/osm_data/osm_data.py

```diff
@@ -0,0 +1,104 @@
+from __future__ import annotations
+
+import hashlib
+import json
+from argparse import ArgumentParser
+from pathlib import Path
+from typing import Any
+from urllib.parse import urlencode
+
+import requests
+import rerun as rr
+import rerun.blueprint as rrb
+
+CACHE_DIR = Path(__file__).parent / "cache"
+if not CACHE_DIR.exists():
+    CACHE_DIR.mkdir()
+
+OVERPASS_API_URL = "https://overpass-api.de/api/interpreter"
+
+# Find some hotels in the area of Lausanne, Switzerland
+# This uses the Overpass API query language: https://wiki.openstreetmap.org/wiki/Overpass_API/Overpass_QL
+QUERY = """
```

Comment:
ultra super nit: ``HOTELS_IN_THE_AREA_QUERY`` (as I wondered below before I read the comment here)

Reply:
good call, fixed